utils/getMultipleCountries.js

```diff
@@ -0,0 +1,35 @@
+const axios = require('axios');
+const numberFormat = require('./numberFormat');
+const exitCountry = require('./exitCountry');
+const to = require('await-to-js').default;
+const handleError = require('cli-handle-error');
+
+module.exports = async (spinner, table, states, countryName, options) => {
+	if (countryName && !states && !options.chart) {
+		const [err, response] = await to(
+			axios.get(`https://corona.lmao.ninja/v2/countries/${countryName}`)
+		);
+		exitCountry(err, spinner, countryName);
+		err && spinner.stopAndPersist();
+		handleError(`API is down, try again later.`, err, false);
+		const thisCountry = response.data;
```

Comment:
Since it's about multiple countries, maybe the name should have "countries"

Reply:
Yes, I agree having countries as variable name seems more meaningful. I have changed that in the latest commit.